app/models/setting/notification.rb

```diff
@@ -3,6 +3,9 @@ def self.default_settings
     [
       self.set('rss_enable', N_('Whether to pull RSS notifications or not'), true, N_('RSS enable')),
       self.set('rss_url', N_('URL to fetch RSS notifications from'), 'https://theforeman.org/feed.xml', N_('RSS URL')),
+      self.set('releases_tracking_enable', N_('Whether to pull release notifications or not'), true, N_('Releases tracking enable')),
+      self.set('releases_track_rc_enable', N_('Whether to include release candidates in the notifications'), false, N_('RC Release tracking enable')),
```

Comment:
We can maybe join these two to one by having a select (`disabled`, `major only` `with RC`)